tests/test_preprocess_pipeline.py

```diff
@@ -49,7 +49,7 @@ def testPhaseFlip(self):
     def testDownsample(self):
         # generate a 3D map with density decays as Gaussian function
         g3d = grid_3d(self.L, dtype=self.dtype)
-        coords = np.array([g3d["x"].flatten(), g3d["y"].flatten(), g3d["z"].flatten()])
+        coords = np.array([g3d["z"].flatten(), g3d["y"].flatten(), g3d["x"].flatten()])
```

Comment:
Same question about why "xy" indexing now rather than "ij".

I was able to run this code with and without switching XYZ -> ZYX. Can't figure out why it's switched?

Reply:
The order doesn't really matter, since the coords are used to generate the test Volume.

The Y points occur in a different order, but iirc the "xy" Z is 'ij' X, vice-versa.  I can revert it if you have strong feelings about it.

Reply:
No no just curious. Thanks!